### DISCUSSION TITLE
What is this built on?

### DISCUSSION THREAD

**Discussion by KaiCodesWithGithub** [rocksdanister/lively#1087] [Q&A]
I've been looking into desktop application development and noticed it had a .net requirement, I've been looking into .net development but don't know which type of application to make, so I chose an application similar to this, can anyone tell me the type of application this is (like asp.net, WPF, UWP, Xamarin...) 

Your contribution is greatly appreciated
Thanks, Kai

**Comment by rocksdanister** [comment]
Lively is a native desktop application using WPF UI framework.

**Reply by KaiCodesWithGithub** [reply]
so .net?
and should i use .net 6.0?

**Reply by rocksdanister** [reply]
>so .net?

Yes
>and should i use .net 6.0?

Sure, why not.

**Reply by KaiCodesWithGithub** [reply]
i want to learn .net and create my own applications but i'm also new to .net, there isn't much on wpf tutorials and thought i could ask someone whos used .net to make an application. i cant figure out how to have multiple pages in a project that you can navigate around

sorry if im bugging you @rocksdanister

**Reply by roxxel** [reply]
> i want to learn .net and create my own applications but i'm also new to .net, there isn't much on wpf tutorials and thought i could ask someone whos used .net to make an application. i cant figure out how to have multiple pages in a project that you can navigate around
> 
> sorry if im bugging you @rocksdanister

That seems off-topic
Dm me in discord I can help u with WPF if u want
Link is in my profile
